client/lerna.json

```diff
@@ -1,7 +1,6 @@
 {
   "version": "0.0.0",
   "npmClient": "yarn",
-  "useWorkspaces": true,
   "packages": [
```

Comment:
I guess it's not part lerna 7 config ?

Reply:
true. the option has been removed and will cause an error if used now.
workspaces are the default and assumed to be in place.